src/main/java/racingcar/game/car/Car.java

```diff
@@ -0,0 +1,82 @@
+package racingcar.game.car;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+public class Car implements Comparable<Car> {
+    private static final String UNNAMED_CAR_FOUND = "이름이 입력되지 않은 자동차가 있습니다.";
+    private static final String NAME_LENGTH_EXCESSIVE = "자동차의 이름은 5글자를 넘을 수 없습니다.";
+    private static final String WHITESPACE_DETECTED = "이름에 공백이 포함된 자동차가 있습니다.";
+    private static final int PROCEED_FLAG = 3;
+    private static final int MAX_NAME_LENGTH = 5;
+    private static final String WHITESPACE = " ";
+
+    private final String name;
+    private final int displacement;
+
+    public Car(String name) {
+        this(name, 0);
+    }
+
+    public Car(String name, int displacement) {
+        validateCarName(name);
+        this.name = name;
+        this.displacement = displacement;
+    }
+
+    public Car conductAction() {
+        if (isAbleToProceed()) {
+            return proceed();
+        }
+        return this;
+    }
+
+    public String showCurrentDisplacement() {
+        return name + " : " + "-".repeat(Math.max(0, displacement));
+    }
+
+    public String displayName() {
+        return name;
+    }
+
+    public boolean isTieWith(Car competitor) {
+        return this.displacement == competitor.displacement;
+    }
+
+    @Override
+    public int compareTo(Car competitor) {
+        return this.displacement - competitor.displacement;
+    }
+
+    private boolean isAbleToProceed() {
+        int randomNumber = Randoms.pickNumberInRange(0, 9);
+        return randomNumber > PROCEED_FLAG;
+    }
+
+    private Car proceed() {
+        return new Car(name, displacement + 1);
+    }
+
+    private void validateCarName(String carName) {
+        validateCarNameFormat(carName);
+        validateCarNameFilled(carName);
+        validateCarNameLength(carName);
+    }
+
+    private void validateCarNameFormat(String carName) {
+        if (carName.isEmpty()) {
+            throw new IllegalArgumentException(UNNAMED_CAR_FOUND);
```

Comment:
```suggestion
        if (StringUtils.isBlank(carName)) {
            throw new IllegalArgumentException(UNNAMED_CAR_FOUND);
```

isBlank 를 통해 " " 또는 Null 인 상태도 잡아줘서 이렇게 바꿔보면 어떨까 의견드립니다.!

Reply:
@ejaee 말씀 주신 내용에 대해 찾아봤는데 isBlank() 는 "", " ", null인 상태를 잡아주네요. 저는 " ", "" 만 잡아준다고 생각하고 있었거든요! 몰랐던 지식이었는데 덕분에 배워갑니다. 감사해요. 😄

Reply:
좋은 기능이라고 생각합니다! 감사합니다!!